src/main/java/edu/cornell/kfs/pdp/businessobject/service/CuACHPayeeSearchService.java

```diff
@@ -0,0 +1,61 @@
+package edu.cornell.kfs.pdp.businessobject.service;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.Pair;
+import org.kuali.kfs.datadictionary.legacy.DataDictionaryService;
+import org.kuali.kfs.fp.businessobject.DisbursementPayee;
+import org.kuali.kfs.fp.document.service.DisbursementVoucherPayeeService;
+import org.kuali.kfs.kim.api.identity.PersonService;
+import org.kuali.kfs.kim.impl.KIMPropertyConstants;
+import org.kuali.kfs.kim.impl.identity.Person;
+import org.kuali.kfs.kns.lookup.Lookupable;
+import org.kuali.kfs.krad.bo.BusinessObjectBase;
+import org.kuali.kfs.krad.util.GlobalVariables;
+import org.kuali.kfs.pdp.PdpConstants;
+import org.kuali.kfs.pdp.PdpKeyConstants;
+import org.kuali.kfs.pdp.businessobject.ACHPayee;
+import org.kuali.kfs.pdp.businessobject.service.ACHPayeeSearchService;
+import org.kuali.kfs.sys.KFSPropertyConstants;
+import org.kuali.kfs.vnd.businessobject.VendorDetail;
+import org.springframework.util.MultiValueMap;
+
+public class CuACHPayeeSearchService extends ACHPayeeSearchService{
+    
+    public CuACHPayeeSearchService(DataDictionaryService dataDictionaryService,
+            DisbursementVoucherPayeeService disbursementVoucherPayeeService, Lookupable vendorLookupable,
+            PersonService personService) {
+        super(dataDictionaryService, disbursementVoucherPayeeService, vendorLookupable, personService);
+    }
+
+    protected Map<String, String> getPersonFieldValues(final Map<String, String> fieldValues) {
```

Comment:
Should the `@Override` annotation be added to this class's methods accordingly?

Reply:
updated